src/Modules/board.js

```diff
@@ -1,20 +1,16 @@
-export default function ScoreBoard(Scores) {
-  const refreshBtn = document.getElementById('refresh');
-  const board = document.querySelector('.board');
-
-  refreshBtn.addEventListener('click', (e) => {
-    e.preventDefault();
-    const scor = JSON.parse(localStorage.getItem('Scores') || []);
-    let boardHtml = '';
-    if (scor) {
-      Scores = scor;
-      Scores.forEach((player) => {
+async function ScoreBoard() {
+  await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/:OsIdfjVZGF6rZmZWBTi6/scores/')
+    .then((res) => res.json())
```

Comment:
- [ ] I will encourage you to take advantage of the arrow function as the function keyword is professionally not recommendable and it also helps javascript hoists in the runtime.